AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -208,6 +209,25 @@ - (void)removePlayerItemObservers:(AVPlayerItem *)playerItem
   [notificationCenter removeObserver:self name:AVPlayerItemNewErrorLogEntryNotification object:playerItem];
 }
 
+- (void)addPlayerObservers:(AVPlayer *)player
+{
+  if (player == nil) {
+    return;
+  }
+
+  [player addObserver:self forKeyPath:kRate options:NSKeyValueObservingOptionNew context:ASVideoNodeContext];
+}
+
+- (void) removePlayerObservers:(AVPlayer *)player
+{
+  @try {
+    [player removeObserver:self forKeyPath:kRate context:ASVideoNodeContext];
+  }
+  @catch (NSException * __unused exception) {
+    NSLog(@"Unnecessary KVO removal");
+  }
```

Comment:
Should I remove the try block? I have very little ObjC/++ experience so if you see anything that you don't like, it can certainly be changed!